src/util.rs

```diff
@@ -152,9 +152,9 @@ pub(crate) fn systemtime_to_offsetdatetime(t: SystemTime) -> time::OffsetDateTim
     match t.duration_since(UNIX_EPOCH) {
         Ok(t) => {
             let tm = time::OffsetDateTime::from_unix_timestamp(t.as_secs() as i64);
-            tm.to_offset(time::offset!(UTC))
+            tm.to_offset(time::UtcOffset::try_current_local_offset().unwrap())
         },
-        Err(_) => time::OffsetDateTime::unix_epoch().to_offset(time::offset!(UTC)),
+        Err(_) => time::OffsetDateTime::unix_epoch().to_offset(time::UtcOffset::try_current_local_offset().unwrap()),
```

Comment:
These changes break webdav on Linux. The `unwrap()` on `try_current_local_offset()`  always fails with `error::IndeterminateOffset`.

Tested on both an x86_64 PC and a ARMv7 device. Both had a timezone (at least`/etc/localtime`) specified.

It seems this issue was already discussed in https://github.com/time-rs/time/issues/296 . I only skimmed that, but it seems to have been introduced a few versions before the used time crate here. There were mentions of vulnerabilities related to it and it seems that current version, don't have this method anymore.

Reply:
I didn't handle the error situation for the machine without the timezone setting.
I modified the unwrap() function to the unwrap_or() function.

Reply:
It's weird since I had a timezone set on my machines. The `chrono` crate (not `time`) picks the timezone up fine.

Commit [6b98eea](https://github.com/miquels/webdav-handler-rs/pull/7/commits/6b98eea7b7b339e349991ededc1b4fd660c31847) resolved the issue.

This is likely just due to the particular version of time. If it is updated, this will need to be fixed to the proper way the crate does it anyway. But seems current dependencies don't favour bumping the version.